diff -Naur gramps-web-api-2.7.0.orig/pyproject.toml gramps-web-api-2.7.0/pyproject.toml
--- gramps-web-api-2.7.0.orig/pyproject.toml	2024-12-31 03:36:48.000000000 -0700
+++ gramps-web-api-2.7.0/pyproject.toml	2025-03-27 16:23:41.563102153 -0600
@@ -16,14 +16,14 @@
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "gramps-desktop[GUI,i18n]==5.2.*",
+    "gramps>=5.2.0",
     "Click>=7.0",
     "Flask>=2.1.0",
     "Flask-Caching>=2.0.0",
     "Flask-Compress",
     "Flask-Cors",
     "Flask-JWT-Extended>=4.2.1,!=4.4.0,!=4.4.1",
-    "Flask-Limiter>=2.9.0",
+    "flask-limiter>=2.9.0",
     "Flask-SQLAlchemy",
     "marshmallow>=3.13.0",
     "waitress",
@@ -36,12 +36,13 @@
     "ffmpeg-python",
     "boto3",
     "alembic",
-    "celery[redis]",
+    "celery",
     "Unidecode",
     "pytesseract",
     "gramps-ql>=0.3.0",
     "object-ql>=0.1.2",
     "sifts>=0.8.3",
+    "torch",
 ]
 
 [project.optional-dependencies]
